packages/pfff/pfff.0.37.5/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Tools and APIs for program analysis, code visualization, refactoring"
+description: """
+pfff is a set of tools and APIs to perform some static analysis,
+dynamic analysis, code visualizations, code navigations, or
+style-preserving source-to-source transformations such as refactorings
+on source code. For now the effort is focused on PHP but there is also
+good support for C, C++, Java, HTML, Javascript, Css, and preliminary
+support for Erlang, Python, C#, Lisp, Scheme, Haskell, OPA, Sql, and
+even TeX. There is also very good support for OCaml and noweb
+(literate programming) so that pfff can be used on the code of pfff
+itself.
+
+For more information see https://github.com/facebook/pfff/wiki/Main"""
+
+maintainer: "yoann.padioleau@gmail.com"
+authors: [ "Yoann Padioleau <yoann.padioleau@gmail.com>" ]
+homepage: "https://github.com/facebook/pfff/wiki/Main"
+bug-reports: "https://github.com/aryx/pfff/issues"
+dev-repo: "git+https://github.com/aryx/pfff.git"
+
+build: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "depend"]
+  [make]
+  [make "opt"]
+]
+install: [
```

Comment:
You need a corresponding `remove` field so that the files are uninstalled, for example when pfff is upgraded (as happens in the CI tests)

```
-> removed   pfff.0.37.5
[WARNING] While removing pfff.0.37.5: not removing files that changed since:
            - lib/pfff-matcher/unit_matcher.cmx
            - lib/pfff-matcher/unit_matcher.cmti
            - lib/pfff-matcher/unit_matcher.cmt
            - lib/pfff-matcher/unit_matcher.cmi
            - lib/pfff-matcher/transforming_fuzzy.cmx
            - lib/pfff-matcher/transforming_fuzzy.cmti
            - lib/pfff-matcher/transforming_fuzzy.cmt
            - lib/pfff-matcher/transforming_fuzzy.cmi
            - lib/pfff-matcher/spatch_fuzzy.cmx
            - lib/pfff-matcher/spatch_fuzzy.cmti
            - lib/pfff-matcher/spatch_fuzzy.cmt
            - lib/pfff-matcher/spatch_fuzzy.cmi
            - lib/pfff-matcher/sgrep_fuzzy.cmx
            - lib/pfff-matcher/sgrep_fuzzy.cmti
```

Reply:
Got it. Thx anil.